database/seeders/ProjectSeeder.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\Dataset;
+use App\Models\NMRium;
+use App\Models\Project;
+use App\Models\Sample;
+use App\Models\Study;
+use App\Models\User;
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Str;
+
+class ProjectSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $password = Str::random();
+        $user = User::factory()->withPersonalTeam()->create(['password' => bcrypt($password)]);
+        $team = $user->personalTeam();
+
+        $project = Project::factory()->create([
+            'team_id' => $team->id,
+            'owner_id' => $user->id,
+        ]);
+
+        $studies = Study::factory(2)->create([
+            'team_id' => $team->id,
+            'owner_id' => $user->id,
+            'project_id' => $project->id,
+            'created_at' => $project->created_at,
+            'updated_at' => $project->updated_at->subMinutes(rand(1, 30)),
```

Comment:
Always updated_at going to be the latest (not sure why the we are subtracting time here?). Also these fields are auto filled by laravel. We don't have to provide them.

Reply:
The point was that if a study is updated, the project will be updated too, while if a project is updated, the study not necessarily is updated, so the update date of a study is earlier or equal to the update date of the project. but as we are unifying the creation and update dates to now, we can modify this